cltk/corpus/latin/corpora.py

```diff
@@ -24,6 +24,11 @@
      'name': 'latin_text_latin_library',
      'location': 'remote',
      'type': 'text'},
```

Comment:
Actually @nelson-liu @theaverageguy 
This `,` causes the build to fail due to `syntax error`.